Netezos/Forging/Local/LocalForge.Parsers.Operations.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Linq;
+using Netezos.Encoding;
+using Netezos.Forging.Models;
+
+namespace Netezos.Forging
+{
+    public partial class LocalParse : IOperationParser
```

Comment:
This file isn't currently being built. I just placed it for stub purposes until I can fill out the unforging operations.